pkg/apiserver/registry/stats/nodelatencystats/rest_test.go

```diff
@@ -28,29 +29,56 @@ import (
 	statsv1alpha1 "antrea.io/antrea/pkg/apis/stats/v1alpha1"
 )
 
+var (
+	fakeClock = testing2.NewFakeClock(time.Now())
```

Comment:
it's better to have a separate clock instance for each test IMO, as some tests may modify the clock